app/src/main/java/org/systers/mentorship/view/fragments/RelationFragment.kt

```diff
@@ -30,6 +35,9 @@ class RelationFragment : BaseFragment() {
     private lateinit var relationViewModel: RelationViewModel
     private val activityCast by lazy { activity as MainActivity }
 
+    //custom alertDialog for verifying the step of cancelling the current relation
+    private val alertDialog by lazy { activity?.let { AlertDialog.Builder(it,R.style.MyCancelDialogTheme) } }
```

Comment:
In PR #141, the dialog was initiated like this: https://github.com/systers/mentorship-android/pull/141/files#diff-5e099f346f07b090bad75bbfbef9163aR23 / Check this please